index.html

```diff
@@ -0,0 +1,136 @@
+  	</p>
+  	<p id="date">
+  	  CONFERENCE & TRAINING MARCH 21 - 24
+  	  <span>KEYNOTE MARCH 22</span>
+    </p>
+   </section>
+  </div>
+  <section id="the-program">
+  	<h2 id="program-heading">Main Program</h2>
+  	<hr>
+  	<ul>
+  		<li>
+	  		<i class="fas fa-server fa-2x"></i>
+	  		<p>Natural Applications</p>
+	  	    <span>
+	  	     	You’ll learn how to use Transformer-based natural language processing models for text classification tasks, such as categorizing documents.
+	  	    </span>			
+  		</li>
+  		<li>
+	  		<i class="fas fa-database fa-2x"></i>
+	  		<p>Data Science</p>
+	  	    <span>
+	  	     	RAPIDS is a collection of data science libraries that allows end-to-end GPU acceleration for data science workflows.
+	  	    </span>			
+  		</li>
+  		<li>
+	  		<i class="fas fa-stream fa-2x"></i>
+	  		<p>Accelerated Computing</p>
+	  	    <span>
+	  	     	The CUDA computing platform enables the acceleration of CPU-only applications to run on the world’s fastest massively parallel GPUs.
+	  	    </span>			
+  		</li> 
+  		<li>
+	  		<i class="fab fa-connectdevelop fa-2x"></i>
+	  		<p>Deep Learning</p>
+	  	    <span>
+	  	     	Discover how deep learning works through hands-on exercises in computer vision and natural language processing.
+	  	    </span>			
+  		</li> 		  		
+  	</ul>
+  	<a rel="noopener noreferrer" href="https://www.nvidia.com/gtc/training/#tabcomponentenhanced290f9bb7_5748_4e21_9a4a_b3a7c9ff75a2-tab-e232" 
+  	   target="_blank" id="program-btn">
+  	      REGISTER FOR GTC
+  	</a>
+
+  	<a 
+  	rel="noopener noreferrer" href="https://www.nvidia.com/gtc/training/#tabcomponentenhanced290f9bb7_5748_4e21_9a4a_b3a7c9ff75a2-tab-e232" 
+  	target="_blank" id="replace-btn">SEE THE WHOLE PROGRAM</a>		
+  </section>
+  <section id="speakers">
+  	
+  </section>
+  <footer>
+	 <section id="partners">
+	 	<h2>Some of Sponsors</h2>
+	 	<hr>
+	 	  <ul>
+		 	  <li><img alt="partner1" src="./images/general/partner1.svg"></li>
+		 	  <li><img alt="partner2" src="./images/general/partner2.svg"></li>
+		 	  <li><img alt="partner3" src="./images/general/partner3.svg"></li>
+		 	  <li><img alt="partner4" src="./images/general/partner4.svg"></li>
+		 	  <li><img alt="partner5" src="./images/general/partner5.svg"></li>
+		 	  <li><img alt="partner6" src="./images/general/partner6.svg"></li>
+		 	  <li><img alt="partner7" src="./images/general/partner7.svg"></li>
+		 	  <li><img alt="partner8" src="./images/general/partner8.svg"></li> 	
+	    </ul>
+	    <a rel="noopener noreferrer" href="https://www.nvidia.com/gtc/sponsors/" 
+	       target="_blank">link to all the sponsors</a>
+	 </section>
+	 <section id="copyright">
+	 	<img alt="local logo" src="./images/general/local-logo.png">
+
+	 	<p>
+	 		<a rel="noopener noreferrer" target="_blank" href="https://twitter.com/search?q=%23GTC22&amp;src=typd">#GTC22</a>
+	 	  USA - United States
+	 	</p>
+	 </section>     	 
+  </footer>	
```

Comment:
- [ ] Hello please make sure to implement the reviewer's suggestion. **You have the latest commit that states you've fixed this although the commit does not have any changes. please kindly make sure to make this change and request a third review. Thanks.

Reply:
the reviewer was referring to the mobile version. please check your mobile version.